.env.example

```diff
@@ -14,6 +21,13 @@ VITE_OISY_ALPHA_WARNING_URL="https://github.com/dfinity/oisy-wallet#status"
 VITE_ERC20_ICP_SYMBOL=ckICP
 VITE_ERC20_ICP_REPO_URL=https://github.com/dfinity/erc20-icp
 VITE_COINGECKO_API_URL=https://api.coingecko.com/api/v3
-VITE_EXCHANGE_DISABLED=false
+VITE_COINGECKO_API_KEY=
+VITE_EXCHANGE_DISABLED=true
+VITE_ETHEREUM_EXPLORER_URL=https://sepolia.etherscan.io
+VITE_BITCOIN_EXPLORER_URL=https://blockstream.info/testnet
+VITE_ICP_EXPLORER_URL=https://dashboard.internetcomputer.org
+VITE_CKBTC_EXPLORER_URL=https://dashboard.internetcomputer.org/bitcoin
+VITE_CKETH_EXPLORER_URL=https://dashboard.internetcomputer.org/ethereum
+VITE_BTC_NETWORK=testnet
```

Comment:
I cannot find a usage for the new variables above except for `VITE_COINGECKO_API_KEY` and `VITE_EXCHANGE_DISABLED`

Reply:
Thanks @AntonioVentilii-DFINITY . Those were left over from the original way to configure the env which were remaining on my side. I cleaned up my env and here adding the one missing.